Remove ZooKeeper from the docs (#10988)
strimzi/strimzi-kafka-operator: documentation/modules/metrics/con_metrics-prometheus-resources.adoc

When you apply the Prometheus configuration, the following resources are created in your Kubernetes cluster and managed by the Prometheus Operator:

> * A `ClusterRole` that grants permissions to Prometheus to read the health endpoints exposed by the Kafka pods, cAdvisor and the kubelet for container metrics.

Reviewer: ```suggestion
* A `ClusterRole` granting Prometheus permissions to read Kafka pod health endpoints and container metrics from cAdvisor and the kubelet.
```
